clients/src/main/java/org/apache/kafka/common/requests/AbstractResponse.java

```diff
@@ -110,7 +111,7 @@ public static AbstractResponse parseResponse(ByteBuffer buffer, RequestHeader re
         return AbstractResponse.parseResponse(apiKey, new ByteBufferAccessor(buffer), apiVersion);
     }
 
-    public static AbstractResponse parseResponse(ApiKeys apiKey, ByteBufferAccessor readable, short version) {
+    public static AbstractResponse parseResponse(ApiKeys apiKey, Readable readable, short version) {
```

Comment:
Why don't you also fix the static method `AbstractResponse parseResponse(ByteBuffer, RequestHeader)`?

Are you planning to fix `RequestHeader` and `ResponseHeader` to use `Readable` instead of `ByteBuffer`?